embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -0,0 +1,38 @@
+package io.embrace.android.embracesdk.arch.schema
+
+internal object EmbType {
```

Comment:
wouldn't be better to use a Sealed Class in case we need to evaluate the type at some point?